app/controllers/nodes_controller.rb

```diff
@@ -16,6 +18,18 @@ def show
     @minion = Minion.find(params[:id])
   end
 
+  def bootstrap
+    salt = Pharos::Salt.new
+    available_roles = %i(master)
+    salt.minions.keys.each do |minion|
```

Comment:
I think we should have a single source of truth. This will make a synchronous request to salt-api to return the available minions (which I think is the equivalent of [```salt '*' grains.items```](https://github.com/saltstack/salt/blob/develop/salt/netapi/rest_cherrypy/app.py#L1156)). If we have the mechanism in place to know when a minion is down (which we should at some point), then maybe we can simply use the available minions from the database.